fbgemm_gpu/hip_kernel/split_tbe_fwd_hip.cpp

```diff
@@ -247,7 +680,7 @@ __device__ void split_tbe_forward_hip_kernel(
     index_t indices[bag_unroll];
     float indice_weights[bag_unroll];
 
-    emb_t emb_data[embedding_dim * bag_prefetch];
+    emb_t emb_data[dword_output_per_row * bag_prefetch];
```

Comment:
This is causing mismatches when we run 64/128/192 etc. embedding dimensions.  I'm not sure the emb_data is supposed to be this small?